piplicenses.py

```diff
@@ -440,11 +440,16 @@ def case_insensitive_partial_match_set_intersect(set_a, set_b):
 
 
 def case_insensitive_partial_match_set_diff(set_a, set_b):
+    """
+    Return items from set_a without case-insensitive partial matches from items in set_b.
```

Comment:
Wrap lines for linter checks.

```suggestion
    Return items from set_a without case-insensitive partial matches
    from items in set_b.
```